src/evaluations/configs.py

```diff
@@ -15,23 +15,32 @@
 import collections
 
 _SketchEstimatorConfig = collections.namedtuple(
-    'SketchEstimatorConfig',
-    ['name', 'sketch_factory', 'estimator', 'noiser'])
+    'EstimatorConfig', ['name', 'sketch_factory', 'estimator', 'sketch_noiser',
+                        'estimate_noiser', 'max_frequency'])
 
 
 # This class exists as a placeholder for a docstring.
 class SketchEstimatorConfig(_SketchEstimatorConfig):
   """A subclass of namedtuple for providing a estimator config to the simulator.
 
   The arguments to the named tuple are as follows:
```

Comment:
Maybe make this the docstring for the class or the __new__ function?

Reply:
I moved the arguments section to the __new__ method and modified the docstring. PTAL. Thanks!